frost-core/src/tests/vectors.rs

```diff
@@ -311,6 +311,10 @@ pub fn check_sign_with_test_vectors<C: Ciphersuite>(json_vectors: &Value) {
 
     // Check that the generated signature matches the test vector signature
     let group_signature = group_signature_result.unwrap();
+    assert_eq!(
+        hex::encode(group_signature.serialize()),
+        hex::encode(signature_bytes.clone()),
+    );
```

Comment:
@zebra-lucky My bad, i think this was a left-over from my debugging phase. You can remove this.

Reply:
Fixed with 1c085ba